### PR TITLE
bugfix: homepage back

### LINKED ISSUE #982: "Category" page on dapp browser should have back button activated
There's a back-arrow icon, but we should have the lower "back" button (on the bottom bar) work also

### PR DESCRIPTION
**Pull request by estebanmino**
**Description**

cLcWaSc0nd

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #982

### CODE REVIEW

File: app/core/AppConstants.js [hunk 1]
```diff
@@ -23,5 +23,6 @@ export default {
 	MM_UNIVERSAL_LINK_HOST: 'metamask.app.link',
 	DAI_ADDRESS: '0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359',
 	HOMEPAGE_URL: 'https://home.metamask.io/',
+	HOMEPAGE_HOST: 'home.metamask.io',
```
Comment (brunobar79) [inline app/core/AppConstants.js]: How about to use getHost with HOMEPAGE_URL instead of declaring 2 almost identical constants?

Comment (brunobar79) [reply app/core/AppConstants.js]: If you do that you can define a local const at the FILE level using 
`const HOMEPAGE_HOST = getHost(HOMEPAGE_URL)`;

**Review by brunobar79 — APPROVED** [review]
💯

File: app/core/AppConstants.js (hunk 1 above)
Comment (estebanmino) [reply app/core/AppConstants.js]: yeah the second option sounds good 👌

**Review by ibrahimtaveras00 — APPROVED** [review]
Looks good on both iOS and Android, QA Passed 👍

**Pull request merged**